.changeset/many-lions-judge.md

```diff
@@ -0,0 +1,5 @@
+---
+"@localyze-pluto/components": patch
```

Comment:
nit: this might not be a patch since it's not a bug fix.. maybe a `minor` ?
https://semver.org/